src/AutomaticVersionRanges.targets

```diff
@@ -0,0 +1,42 @@
+<Project>
+
+  <PropertyGroup>
+    <AutomaticVersionRangesEnabled Condition="'$(AutomaticVersionRangesEnabled)' == '' And '$(Configuration)' == 'Debug'">false</AutomaticVersionRangesEnabled>
+    <AutomaticVersionRangesEnabled Condition="'$(AutomaticVersionRangesEnabled)' == '' And '$(IsPackable)' == 'false'">false</AutomaticVersionRangesEnabled>
+    <AutomaticVersionRangesEnabled Condition="'$(AutomaticVersionRangesEnabled)' == '' And '$(ManagePackageVersionsCentrally)' == 'true'">false</AutomaticVersionRangesEnabled>
+    <AutomaticVersionRangesEnabled Condition="'$(AutomaticVersionRangesEnabled)' == ''">true</AutomaticVersionRangesEnabled>
+  </PropertyGroup>
+
+ <UsingTask TaskName="ConvertToVersionRange" TaskFactory="RoslynCodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.Core.dll" >
+  <Task>
+    <Code Source="$(MSBuildThisFileDirectory)ConvertToVersionRange.cs" />
```

Comment:
I would consider finding a name for a subfolder to put this file and the C# file in. The more I think about it the more I think people are going to be very confused seeing a C# file in the src folder, and may try to move/remove it, leading to resync PRs. I know `targets` is a poor choice because of https://github.com/Particular/NServiceBus.RabbitMQ/tree/master/src/targets but what about `packaging` or `packaging-targets`? Long term the strong name keys could also move into such a folder to have them more out of the way, but that's obviously a much bigger change that would also require cleanup. Something to think about, anyway.

Reply:
There's no reason that any future targets added in the same manner would be related to packaging concerns, so I wouldn't want `packaging` in the name. Perhaps `msbuild` or `msbuild-targets` ?

Reply:
I like `msbuild`